src/plugins/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -91,7 +91,7 @@ export const EditIndexPattern = withRouter(
         .then((resp) => {
           setRelationships(resp.relations.map((r) => ({ ...r, title: r.meta.title! })));
         });
-    }, [savedObjectsManagement, indexPattern, allowedTypes]);
+    }, [savedObjectsManagement, indexPattern, allowedTypes, indexPattern]);
```

Comment:
I think adding indexPattern once should be sufficient?
```suggestion
    }, [savedObjectsManagement, indexPattern, allowedTypes]);
```